src/template.html

```diff
@@ -139,39 +139,38 @@
           />
         </div>
         <ul
-          class="flex overflow-hidden gap-4 font-semibold text-2xl rounded-lg px-8 py-6 w-fit text-slate-100 bg-black/30 backdrop-blur-md sm:hidden"
+          class="flex items-center overflow-hidden sm:gap-1 gap-4 font-semibold sm:text-xl text-2xl rounded-2xl sm:mx-4 px-8 py-2 sm:px-2  w-fit text-slate-100 bg-black/30 backdrop-blur-md"
         >
-          <li class="">
+          <li>
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link flex active p-6 pr-4 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Games
               <span id="game-counter"></span>
             </a>
           </li>
-          <li class="">
+          <li>
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link flex p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Trailers</a
             >
           </li>
           <li class="">
             <a
-              class="p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
-              href="#"
+              class="nav-link p-6 hover:bg-white hover:text-gamify-black hover:rounded-xl duration-300"
+              href="#content"
               >Contact</a
             >
           </li>
         </ul>
       </nav>
     </header>
     <main>
-      <section id="games-display">
-        <h2 class="text-[0px] visually-hidden">List of games</h2>
-        <div class="game-list mt-12 grid gap-4">
-          <!-- Generated -->
-        </div>
+      <section id="content">
+        <h2 class="text-[0px] visually-hidden">List of games, trailers or Contact</h2>
+
+        <!-- Generated -->
       </section>

```

Comment:
### Great use of the single-page application approach

I am loving the fact that you introduced a lot of features herein with the game trailer videos and more fascinating is how you did that with the single-page application approach. Good work!